MANIFEST.in

```diff
@@ -0,0 +1,6 @@
+include LICENSE
+
+recursive-include test *.py
+
+exclude tox.ini
```

Comment:
Running ``python setup.py test`` will run the test suite directly, without installing tox. This is a good thing - someone doing that is verifying tests on their system with their Python, not against the entire test suite. That being said, there is no harm in including ``tox.ini``. Which way do you want to go?